docs/automated-testing/ui-testing/teams-tests.md

```diff
@@ -0,0 +1,240 @@
+    1. Manually install Microsoft Teams from Google Playstore on the device. 
+
+####  b) Steps to set up Appium
+- Install `appium`:
+    1. Download NodeJs, if it is not already installed on your machine: [Download | Node.js (nodejs.org)](https://nodejs.org/en/download)
+    1. Install Appium globally: [Install Appium - Appium Documentation](https://appium.io/docs/en/2.0/quickstart/install/)
+    1. Install the UiAutomator2 driver: [Install the UiAutomator2 Driver - Appium Documentation](https://appium.io/docs/en/2.0/quickstart/uiauto2-driver/). Go through the `Set up Android automation requirements` in the documentation, to make sure you have set up everything correctly.
+
+        > [Here](https://appium.io/docs/en/2.0/intro/drivers/) you can read more about Appium Drivers.
+    1. Start appium server by running `appium` command in a command prompt.
+####  Useful commands:
+- To list the emulators that you have previously created, without opening Android Studio:
+    ```cli
+    emulator -list-avds
+    ```
+- To start an emulator, without opening Android Studio:
+    ```cli
+    emulator -avd {name-of-emulator}
+    ```
+- To get the name of your AVD, assuming your emulator is running, you can run:
+    
+    ```cli
+    adb devices
+    ```
+    You should see a response with *List of devices attached*, where your running device will be listed.
+
+### How to set up an Azure VM for running mobile tests in a pipeline
+
+### 1. VM configuration
+This approach essentially hosts a virtual device in a virtual machine. In order to be able to set up the emulator (Android Virtual Device) in an Azure VM, the VM should support [nested virtualization](https://azure.microsoft.com/en-us/blog/nested-virtualization-in-azure/). 
+
+Azure VM configuration which, at the time of writing the documentation, worked successfully with AVD and appium:
+- Operating system: Windows (Windows-10 Pro)
+- VM generation: V1
+- Size: Standard D4ds v5 16 GiB memory
+
+### 2. Enable connection from outside to Appium server on the VM 
+> Note: By default appium server runs on port 4723. The rest of the steps will assume that this is the port where your appium server runs.
+
+In order to be able to reach appium server which runs on the VM from outside, the following steps are needed:
+1. Create an [Inbound Rule](https://learn.microsoft.com/en-us/windows/security/operating-system-security/network-security/windows-firewall/create-an-inbound-port-rule) for port 4723 from within the VM.
+1. Create an Inbound Security Rule in the NSG (Network Security Group) of the VM to be able to connect from that IP address to port 4723.
+    - Find out the IP of the machine on which the tests will run on.
+    - Replace the *Source IP Address* with the IP of your machine.
+
+### 3. Install Android Studio and create AVD inside the VM
+1. Follow the instructions from steps a) and b) in **How to run mobile end to end tests on a Windows machine** section above.
```

Comment:
```suggestion
### Installing Android Studio and create AVD inside the VM

1. Follow the instructions from steps a) and b) in **How to run mobile end to end tests on a Windows machine** section above.
```
```suggestion
### 3. Install Android Studio and create AVD inside the VM
1. Follow the instructions from steps a) and b) in **How to run mobile end to end tests on a Windows machine** section above.
```
```suggestion
### Install Android Studio and create AVD inside the VM

1. Follow the instructions from steps a) and b) in **How to run mobile end to end tests on a Windows machine** section above.
```

Reply:
NOTE: This section does not exist above... Point to the correct section and link with the #link